### ISSUE TITLE
Any chance of using a fix argument

### ISSUE THREAD

**Issue by spicemix** [andrewbrey/npm-global-audit#1]
`npm audit fix` and `npm audit fix --force` are what npm offers to fix these problems without waiting for a new release with new deps over npm. While I know ` npx npm-global-audit --auditor npm ` lets us use npm for the global audit, I don't see how to pass a `fix` argument in to npm. Is it possible with your script?

Thanks for your work on this, an audit is better than nothing, but a fix is best of all (and if the fix breaks things, we can roll back).

Update: I was led to this repo from [this QA](https://stackoverflow.com/questions/52538773/how-to-npm-audit-global-packages). In the answers there is a solution where you make a package.json and lockfile and run audit fix yourself. This didn't really do anything but wreck the usual `npm-check -ug` with a million little global packages to update, which I'm not going to try, so maybe fixing this isn't a good idea after all. Anyway, leaving this open for your thoughts on actually fixing global deps issues yourself. Thanks!

**Comment by andrewbrey** [comment]
@spicemix Thanks for the issue, and for using my module. Being able to pass `--fix` to the audit call wouldn't be too hard to add, but I do not know when I can get to it. I am happy to review and potentially merge in a PR if you want to take a stab at adding this. The basic idea would be simply accepting an additional flag in the Oclif command, then threading that down to the child process call for an audit as appropriate. Cheers!

**Comment by andrewbrey** [comment]
@spicemix I gave this a bit more of a think now that it's not the middle of the night, and I realized that while such a command (`npm audit fix --force`) could be issued by this utility, it would only be issued against the `node_modules` contained in the temporary directory project created to simulate the globally installed `node_modules`. Running `fix` in there would not have any impact at all on your globally installed modules, and in fact, I feel that introducing this would cause confusion to users who assume

> oh, running `npx npm-global-audit -a npm --fix` is going to solve my reported audit issues for my global packages

when in reality, it won't do anything to them, at least, not using the tools provided to us by the `npm` cli. If you really wanted to "fix" the global modules following an audit, you would have to go into the global install files and mess about with the files yourself, simulating what `npm` can do for local packages....there are so many things that could go wrong there, and that's complexity that I don't wish to add to this utility.

Unfortunately, this request isn't something that can be easily added to this tool, and I'm skeptical that it can be done _at all_ (reliably and without side effects) without the implementation actually being a part of the npm cli in the first place - at which point this utility would no longer be needed.

I've closed the issue for this reason - thank you for writing up the issue, but this is not something that will, or even can, be added to `npm-global-audit`. Cheers!
